turtlebot3c-bringup-snap/snap/local/core_launcher.sh

```diff
@@ -8,8 +7,8 @@ if [[ -z "${TURTLEBOT3_MODEL}" ]]; then
     export TURTLEBOT3_MODEL="$(snapctl get turtlebot3-model)"
 fi
 
-if [[ -z "${TURTLEBOT3_SIMULATION}" ]]; then
-    TURTLEBOT3_SIMULATION="$(snapctl get simulation)"
+if [[ -z "${SIMULATION}" ]]; then
+    SIMULATION="$(snapctl get simulation)"
 fi
 
-$SNAP/roslaunch turtlebot3c_bringup turtlebot3c_bringup.launch simulation:=$TURTLEBOT3_SIMULATION
+ros2 launch turtlebot3_bringup robot.launch.py use_sim_time:=$SIMULATION
```

Comment:
This launch is going to launch the whole tb3 stack right? https://github.com/ROBOTIS-GIT/turtlebot3/blob/humble-devel/turtlebot3_bringup/launch/robot.launch.py#L88

Isn't it accessing USB and serial port?

Reply:
yes, that part is gonna not work and fail and just the robot_state_publisher is gonna stay up. Do you think we should still have a higher level file that avoids launching stuff that won't work?